src/test/java/net/sf/marineapi/nmea/parser/SentenceFactoryTest.java

```diff
@@ -233,6 +233,7 @@ public void testRegisterParserWithAlternativeBeginChar() {
 		assertTrue(s instanceof VDMParser);
 		instance.unregisterParser(VDMParser.class);
 		assertFalse(instance.hasParser("VDM"));
+		instance.reset();

```

Comment:
I think this should be placed in `@After` method to ensure it's done after every test.